### PR TITLE
Haiku localization en, de, fr, it

### PR DESCRIPTION
**Pull request by janus2**
* Semi automatic conversion...

### CODE REVIEW

**Comment by janus2** [comment]
I kept the Constant strings for reference, if this is correct we can remove that section.

File: Main/locales/de.catkeys [hunk 1]
```diff
@@ -0,0 +1,92 @@
+1	English	application/x-vnd.pecora-PecoRename	586651114
+Well...	PecoApp		Äh...
+Errors occurred.	PecoApp		Es sind Fehler aufgetreten.
+About PecoRename	PecoApp		Über...coRename
+UPPERCASE	Renamer_UpperLower		GROSSBUCHSTABEN
+Thank you :-)	PecoApp		Dankeschön :-)
+Remove	Renamer_Remove		Entfernen
+I expect some problems with double used file names.\n\nShould I still start with the renaming?	PecoApp		Es wird voraussichtlich Probleme mit doppelt vorhandenen Dateinamen geben.\n\nSoll trotzdem mit dem Umbenennen begonnen werden?
+Uppercase / lowercase:	Renamer_Extension		Groß- REN_UPPERLOWER Kleinschreibung:
```
Comment (humdingerb) [inline Main/locales/de.catkeys]: The REN_UPPERLOWER should be a "&".

File: Main/locales/de.catkeys [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+Ok	Fenster		Ok
+from the back (right)	Renamer_InsertReplace		von hinten (rechts)
+Continue without messages	PecoApp		Fortfahren_WO_MSG
+Status: 	functions		Status: 
+At position:	Renamer_InsertReplace		An Position:
+Mode:	MainView		Modus:
+Uppercase / lowercase	Renamer_UpperLower		Groß- REN_UPPERLOWER Kleinschreibung
```
Comment (humdingerb) [inline Main/locales/de.catkeys]: Here too, the REN_UPPERLOWER should be a "&".

File: Main/locales/de.catkeys [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+Errors occurred.	PecoApp		Es sind Fehler aufgetreten.
+About PecoRename	PecoApp		Über...coRename
+UPPERCASE	Renamer_UpperLower		GROSSBUCHSTABEN
+Thank you :-)	PecoApp		Dankeschön :-)
+Remove	Renamer_Remove		Entfernen
+I expect some problems with double used file names.\n\nShould I still start with the renaming?	PecoApp		Es wird voraussichtlich Probleme mit doppelt vorhandenen Dateinamen geben.\n\nSoll trotzdem mit dem Umbenennen begonnen werden?
+Uppercase / lowercase:	Renamer_Extension		Groß- REN_UPPERLOWER Kleinschreibung:
+Ok	Fenster		Ok
+from the back (right)	Renamer_InsertReplace		von hinten (rechts)
+Continue without messages	PecoApp		Fortfahren_WO_MSG
```
Comment (humdingerb) [inline Main/locales/de.catkeys]: "Fortfahren_WO_MSG" isn't quite right... Probably: "Ohne Meldungen fortfahren".

File: Main/locales/fr.catkeys [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+Errors occurred.	PecoApp		Erreurs détectées.
+About PecoRename	PecoApp		A propos...coRename
+UPPERCASE	Renamer_UpperLower		MAJUSCULE
+Thank you :-)	PecoApp		Merci :-)
+Remove	Renamer_Remove		Supprimer
+I expect some problems with double used file names.\n\nShould I still start with the renaming?	PecoApp		Risque d'erreur du à l'utilisation multiple d'un nom de fichier Voulez-vous procéder au renommage?
+Uppercase / lowercase:	Renamer_Extension		Majuscule / Minuscule:
+Ok	Fenster		Ok
+from the back (right)	Renamer_InsertReplace		en fin (à droite)
+Continue without messages	PecoApp		Continuer_WO_MSG
```
Comment (humdingerb) [inline Main/locales/fr.catkeys]: "Continuer_WO_MSG"... I don't speak French, but Google's  "Continuer sans messages" sounds correct.

**Comment by janus2** [comment]
Spelled your nick wrong in the commit title... WT HELL
Yes, you can do whatever you want. I am only afraid that nobody will translate the new strings.
Can you wait a fews days? I think someone have removed a few checks, because the errors are not triggered in different cases. If I have time I want to check (a few messages could change) and I will finish cleanup the code.

**Comment by humdingerb** [comment]
Sure. Just say the word when you're done for now and I'll slo-mo into action.

**Pull request merged**